src/common/zscore/cgmzscore/src/table.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+
+class Table:
+
+    def __init__(self, table_file: str):
+        self.table_file = table_file
+
+    def loadTable(self) -> None:
+        open_table = open(self.table_file, 'r')
```

Comment:
`open_table` sounds like a function because it starts with a verb. I suggest:

```suggestion
        table_file = open(self.table_file, 'r')
```